BlueprintUI/Sources/BlueprintView/BlueprintView.swift

```diff
@@ -364,7 +364,10 @@ public final class BlueprintView: UIView {
         setNeedsLayout()
     }
 
+    @MainActor
     private func updateViewHierarchyIfNeeded() {
+        MainActor.preconditionIsolated("updateViewHierarchyIfNeeded must only execute on the main actor.")
```

Comment:
TIL, neat!

Reply:
Thought: Should we add something to the error that this is _caller error_, not blueprint error?

Reply:
> TIL, neat!

Same! haha. [Slack](https://square.slack.com/archives/C03GDC335/p1730221000655359)

Though it looks like it was only backported sometime between Xcode 15.1 and 15.4 (and we're still using 15.1 in Blueprint CI fml)

```
/Users/runner/work/Blueprint/Blueprint/BlueprintUI/Sources/BlueprintView/BlueprintView.swift:369:19: error: 'preconditionIsolated(_:file:line:)' is only available in iOS 17.0 or newer
```

Reply:
> Thought: Should we add something to the error that this is caller error, not blueprint error?

I've expanded the error message a bit to encourage folks to investigate the callers if they ever hit this.